### DISCUSSION TITLE
Details are incorrect ! Connection not established.

### DISCUSSION THREAD

**Discussion by Joao-Monticelli** [uvdesk/community-skeleton#757] [General]
Estou configurando o UVDESK com XAMMP em uma nova maquina e estou trancado na parte de conexão com o banco, tenho outras aplicaçãoes rodando normalmente nele, somente o UVDESK esta com esse problema.
O devo fazer?
![erro uvdesk](https://github.com/uvdesk/community-skeleton/assets/121101215/7655a3c4-b0cd-4f23-898b-f6c26e918508)

**Comment by papnoisanjeev** [comment]
@Joao-Monticelli
We have released a new version(v1.1.7) you can try.
Also this problem may be due to [this](https://github.com/uvdesk/community-skeleton/issues/364#issuecomment-780486976), please check.
